packages/adapter-neo4j/src/index.ts

```diff
@@ -113,20 +111,20 @@ export interface Neo4jOptions extends Session {}
  *
  * ```cypher
  * CREATE CONSTRAINT account_provider_composite_constraint IF NOT EXISTS
- * ON (a:Account) ASSERT (a.provider, a.providerAccountId) IS NODE KEY;
+ * FOR (a:Account) REQUIRE (a.provider, a.providerAccountId) IS NODE KEY;
  *
  * CREATE INDEX account_provider_composite_index IF NOT EXISTS
  * FOR (a:Account) ON (a.provider, a.providerAccountId);
  *
  * CREATE CONSTRAINT verification_token_composite_constraint IF NOT EXISTS
- * ON (v:VerificationToken) ASSERT (v.identifier, v.token) IS NODE KEY;
+ * FOR (v:VerificationToken) REQUIRE (v.identifier, v.token) IS NODE KEY;
  *
  * CREATE INDEX verification_token_composite_index IF NOT EXISTS
  * FOR (v:VerificationToken) ON (v.identifier, v.token);
  * ```
  */
-export function Neo4jAdapter(session: Session): Adapter {
-  const { read, write } = client(session)
+export function Neo4jAdapter(driver: Driver): Adapter {
```

Comment:
Are you referring to v4 of next-auth or the neo4j-driver package?

This would be a breaking change for the Neo4jAdapter, as sessions would no longer be accepted arguments.